src/output/grid.rs

```diff
@@ -56,21 +56,28 @@ impl<'a> Render<'a> {
                     0
                 };
 
+            let mut additionnal_chars = 0;
```

Comment:
Can we do something like this?
```Rust
let offset = if file.name.contains(' ') { 2 } else { 0 }

Reply:
Yeah ok works for me if you don't feel like we'll need the offset for something else. I'll fix it quickly when I get home.

Reply:
Actually now that I look at it. Maybe we should do 
```Rust
let offset = if file.name.contains(' ') { 3 } else { 1 }
// .....
ShowIcons::Automatic(spacing)) => filename.bare_width() + classification_width + offset + (spacing as usize)
// etc.. 
```
And then we can remove the hardcoded ` + 1` value

Reply:
Well the +2 is also needed for the hyperlink only case